node/src/actors/chain_manager/mod.rs

```diff
@@ -2008,6 +2011,20 @@ impl ChainManager {
         // Ice the invalid blocks' batch sender
         self.ice_peer(sender);
     }
+
+    /// Return the value of the version field for a block in this epoch
+    fn tapi_votes_mask(&self, epoch: Epoch) -> u32 {
```

Comment:
Let's run away from selling this as a voting / governance mechanism.

```suggestion
    fn tapi_signals_mask(&self, epoch: Epoch) -> u32 {
```